cli/cmd/proxy/read/config.go

```diff
@@ -396,45 +396,154 @@ func parseSecrets(rawCfg map[string]interface{}) ([]Secret, error) {
 	return secrets, nil
 }
 
-func formatFilters(filterChain filterChain) (filters []string) {
+func formatFilters(filters []filter) (formatted []string) {
```

Comment:
We avoid using named return parameters as it could be confusing. In this case, given we are explicitly returning `formatted` there is no need to keep this.
```suggestion
func formatFilters(filters []filter) []string {
```